tests/test.c

```diff
@@ -647,3 +647,27 @@ void check_param_range(char* dev_name, uint64_t uid, char* param_name, param_typ
     }
     print_pass();
 }
+
+// Returns if the latency is neither too high nor negative
+void check_latency(char* dev_name, uint64_t uid, char* param_name, int32_t upper_bound_latency, int32_t start_time) {
+    uint8_t dev_type = device_name_to_type(dev_name);
+    device_t* dev = get_device(dev_type);
```

Comment:
Need to check for errors as a result of both of these functions.

Reply:
Oh right! Good catch!